tests/api/routes/projects.test.js

```diff
@@ -54,6 +54,53 @@ describe('tests for projects route', () => {
       });
   });
 
+  test('Creating project with correctly shaped project returns 200', async (done) => {
+    const payload = {
+      name: 'Test project',
+      description: '',
+      createdDate: '',
+      lastModified: '',
+      uuid: 'project-1',
+      experiments: [],
+      lastAnalyzed: null,
+      samples: [],
+    };
+
+    request(app)
+      .post('/v1/projects/someId')
+      .send(payload)
+      .expect(200)
+      .end((err) => {
+        if (err) {
+          return done(err);
+        }
+        return done();
+      });
+  });
+
+  test('Creating project with invalid shape project returns 400', async (done) => {
+    const payload = {
+      name: 'Test project',
+      description: '',
+      createdDate: '',
+      lastModified: '',
+      uuid: 'project-1',
+      experiments: [],
+      lastAnalyzed: null,
```

Comment:
This looks quite valid. Maybe you forgot to make it invalid and modify the 200 into 400?

Reply:
that's what happened, I was looking into the mocks yet, they are returning mocked invalids which don't really help to test the api validation in any way (it should be caught before it reaches the mock by the schema validator but the vlaidation isn't really restrictive)

Reply:
see https://github.com/biomage-ltd/api/pull/231#discussion_r723576946